src/main/java/com/epam/aidial/core/ProxyContext.java

```diff
@@ -127,6 +128,12 @@ public Future<Void> respond(HttpStatus status, String body) {
         return response.setStatusCode(status.getCode()).end(body);
     }
 
+    public Future<Void> respond(Throwable error) {
+        return error instanceof HttpException exception
+                ? respond(exception.getStatus(), exception.getMessage())
+                : respond(HttpStatus.INTERNAL_SERVER_ERROR, error.getMessage());
```

Comment:
It is a bit unsafe to send error.getMessage(). It would nice to provide a fallback message to use in this case. respond(Throwable error, String fallbackMessage)

Reply:
Done.